tests/security/swtpm/swtpm_verify.pm

```diff
@@ -17,19 +17,14 @@ use version_utils 'is_sle';
 sub run {
     select_serial_terminal if !(get_var('MACHINE') =~ /RPi4/);
     if (is_sle('=15-SP6')) {
-        record_soft_failure('SKIPPING TEST - poo#179729');
+        record_soft_failure('SKIPPING TEST; poo#179729 bsc#1242989');
         return;
     }
-    my $vm_type = 'legacy';
-    $vm_type = 'uefi' if get_var('UEFI');
+    my $vm_type = get_var('UEFI') ? 'uefi' : 'legacy';
```

Comment:
since you are no interested in the actual variable value, I'd suggest to use `check_var`